.github/workflows/continuous-integration-workflow.yml

```diff
@@ -1,6 +1,12 @@
 name: build
 
-on: [pull_request, push]
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
 
```

Comment:
this seems like it was useful for running on the actual relevant branch and not just master regardless

Reply:
This is on purpose: https://github.community/t/how-to-trigger-an-action-on-push-or-pull-request-but-not-both/16662/2

Reply:
this link should be added as comment then, imo: https://github.community/t/how-to-trigger-an-action-on-push-or-pull-request-but-not-both/16662

Reply:
ok